nuxt.config.ts

```diff
@@ -25,6 +25,9 @@ export default defineNuxtConfig({
     },
     highlight: false,
   },
+  tailwindcss: {
```

Comment:
Hmmm, It adds 4kb to the client bundle. What benefit does it have ?

https://tailwindcss.nuxtjs.org/tailwind/config#tailwindconfig